### DISCUSSION TITLE
Date & time picker and text editor

### DISCUSSION THREAD

**Discussion by seabasss** [livewire/flux#144] [Feature Requests] [answered]
I didn’t find a comments section… Are there any components for date and time pickers and WYSIWYG/Markup editors on the roadmap?

Thanks!

**Comment by patrykszady** [comment]
Yes, date and timepicker have been mentioned in an email with a roadmap. I think right now it's best to focus on making the existing components work as intended before adding anything new.

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by kyranb** [comment, marked as answer]
There's another discussion [here](https://github.com/livewire/flux/discussions/769)

**Reply by joshhanley** [reply]
Thanks! Going to close this in favour of #769 for the date picker (and we now have a text editor).
